ajax/lib/Dropify/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "dropify",
+  "version": "0.2.1",
+  "filename": "js/dropify.min.js",
+  "description": "Override your input files with style.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/JeremyFagis/dropify.git"
+  },
+  "author": "Jeremy FAGIS",
+  "homepage": "http://jeremyfagis.github.io/dropify/",
+  "npmName": "dropify",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "css/dropify*",
+        "js/*"
```

Comment:
I prefer to use `"**/*"` to grab all the files under dist folder, including the fonts.